easybuild/easyconfigs/c/Clang-Python-bindings/Clang-Python-bindings-8.0.0-GCCcore-8.2.0-Python-2.7.15.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'Tarball'
+
+name = 'Clang-Python-bindings'
+version = '8.0.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://clang.llvm.org'
+description = """Python bindings for libclang"""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ["https://llvm.org/releases/%(version)s"]
+sources = ['cfe-%(version)s.src.tar.xz']
+checksums = ['084c115aab0084e63b23eee8c233abb6739c399e29966eaeccfc6e088e0b736b']
+
+dependencies = [
+  ('Clang', version),
+  ('Python', '2.7.15')
```

Comment:
@rokzitko Please fix indent, each indent level should be 4 spaces.

Same below for `sanity_check_paths`.